content/en/examples/admin/konnectivity/konnectivity-server.yaml

```diff
@@ -8,12 +8,13 @@ spec:
   hostNetwork: true
   containers:
   - name: konnectivity-server-container
-    image: registry.k8s.io/kas-network-proxy/proxy-server:v0.0.32
+    image: registry.k8s.io/kas-network-proxy/proxy-server:v0.0.37
     command: ["/proxy-server"]
     args: [
             "--logtostderr=true",
             # This needs to be consistent with the value set in egressSelectorConfiguration.
             "--uds-name=/etc/kubernetes/konnectivity-server/konnectivity-server.socket",
+            "--delete-existing-uds-file=true",
```

Comment:
nit: do you need `=true`, or can the flag just be `--delete-existing-uds-file`?

Reply:
I verified that the `=true` is not needed.